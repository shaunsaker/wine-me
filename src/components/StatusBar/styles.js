import { StyleSheet } from 'react-native';

import styleConstants from '../../styleConstants';

const STATUS_BAR_HEIGHT = 22;

const styles = StyleSheet.create({
  container: {
    position: 'absolute',
    top: 0,
    left: 0,
    right: 0,
    zIndex: 100,
  },
  statusBar: {
    height: styleConstants.helpers.isIphoneX ? STATUS_BAR_HEIGHT + 24 : STATUS_BAR_HEIGHT,
  },
});

export default styles;
