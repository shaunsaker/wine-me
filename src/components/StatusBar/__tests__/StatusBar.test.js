import React from 'react';
import renderer from 'react-test-renderer';

import StatusBar from '..';

describe('StatusBar', () => {
  describe('renders', () => {
    it('renders with minimum required props', () => {
      expect(renderer.create(<StatusBar />)).toMatchSnapshot();
    });
  });
});
