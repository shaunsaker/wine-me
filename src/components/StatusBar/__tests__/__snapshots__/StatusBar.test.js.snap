// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusBar renders renders with minimum required props 1`] = `
<View
  style={
    Array [
      Object {
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
        "zIndex": 100,
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "height": 22,
        },
        Object {
          "backgroundColor": "#FFFFFF",
        },
      ]
    }
  />
</View>
`;
